mobile-app/app/api/transaction/dfi_converter.ts

```diff
@@ -38,21 +38,16 @@ export async function dfiConversionSigner (account: WhaleWalletAccount, amount:
 }
 
 export function dfiConversionCrafter (amount: BigNumber, mode: ConversionMode, onBroadcast: () => any, submitButtonLabel?: string): DfTxSigner {
+  const [symbolA, symbolB] = mode === 'utxosToAccount' ? ['UTXO', 'DFI'] : ['DFI', 'UTXO']
   return {
     sign: async (account: WhaleWalletAccount) => await dfiConversionSigner(account, amount, mode),
     title: translate('screens/ConvertConfirmScreen', 'Converting DFI'),
-    description: translate('screens/ConvertConfirmScreen', 'Converting {{amount}} {{symbolA}} to {{symbolB}}', {
-      amount: amount.toFixed(8),
-      ...(mode === 'utxosToAccount'
-        ? {
-          symbolA: 'UTXO',
-          symbolB: 'Token'
-        }
-        : {
-          symbolA: 'Token',
-          symbolB: 'UTXO'
-        })
-    }),
+    drawerMessages: {
+      preparing: translate('screens/OceanInterface', 'Preparing to convert…'),
+      waiting: translate('screens/OceanInterface', 'Converting {{symbolA}} to {{symbolB}}', { symbolA, symbolB }),
+      complete: translate('screens/OceanInterface', 'Conversion completed')
```

Comment:
![image](https://user-images.githubusercontent.com/13292662/183342252-4747fde1-9923-4eb9-b4e9-411a42299cbf.png)

Reply:
We are using more contextual messages for txn drawer, So above message is already confirmed